.github/actions/setup-binary-builds/action.yml

```diff
@@ -45,16 +45,41 @@ runs:
           ref: ${{ inputs.ref }}
           submodules: ${{ inputs.submodules }}
           path: ${{ inputs.repository }}
-      - name: Set release CHANNEL
+      - name: Log Available Webhook Fields
         shell: bash -l {0}
         run: |
+          echo "ENV VARS"
+          echo "${GITHUB_REF_NAME}"
+          echo "${GITHUB_REF}"
+          echo "${GITHUB_BASE_REF}"
+
+          echo "GITHUB PROVIDED"
+          echo "${{ github.ref_name }}"
+          echo "${{ github.event.ref }}"
+          echo "${{ github.ref }}"
+          echo "${{ github.base_ref }}"
+      - name: Set default CHANNEL
+        shell: bash -l {0}
+        run: |
+          echo "CHANNEL=nightly" >> "${GITHUB_ENV}"
+      - name: Set CHANNEL for tagged pushes
+        if: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/') }}
```

Comment:
2 Questions here:

1. Are `github.event.ref` and `github.ref` the same? They seem to be used interchangeably (like in the next step below). However `github.event.ref` for this PR is null, whereas `github.ref` is `refs/pull/2516/merge`.
2. Do these propagate correctly from caller/reusable workflows? Can do a quick test to find out, but if neither these nor the corresponding env vars are, then we would need to pass a bunch of these values from the caller.

Reply:
1. They might / might not be, if we're seeing discrepancies then let's just go for the one that yields us the best results to remain consistent(see [docs](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context))
![Screenshot 2023-02-09 at 8 40 29 AM](https://user-images.githubusercontent.com/1700823/217879777-cc1628be-82cd-445b-8165-7552154d8406.png)
2. The github context _should_ propagate no matter where we're referencing it

Reply:
Did a quick experiment and looks like the context is propagating. The workflow here has the same conditions as the old M1 workflow: https://github.com/pytorch/vision/blob/09030c02ae88c87bbeafded72d27df7ea4213196/.github/workflows/build-m1-binaries.yml#L29, so I hope this will work smoothly. But the null `github.event.ref` is still worrisome and per the docs `github.ref` seems more correct.